modules/imgproc/test/test_resize_bitexact.cpp

```diff
@@ -152,4 +152,33 @@ TEST(Resize_Bitexact, Linear8U)
         }
 }
 
+static int checkNearest8U_vsNonExact(const Mat& src)
+{
+    Mat dstExact; cv::resize(src, dstExact, Size(), 2, 1, INTER_NEAREST_EXACT);
+    Mat dstNonExact; cv::resize(src, dstNonExact, Size(), 2, 1, INTER_NEAREST);
+
+    return cv::norm(dstExact, dstNonExact, NORM_INF);
```

Comment:
Technically it's not a correct verification. If the scaled point falls exactly between two adjacent source points that have sufficiently different brightness(some picture edge)it could be solved to either one of them. Such choice of the point will be minimal possible(and therefore acceptable) difference from NN resize point of view, but simultaneously it could cause almost maximum brightness difference and fail such verification. So the check is correct only for highly blurred images without sharp edges

Reply:
I think that values in `src_color`, `src_gray`, `mat_color` and `mat_gray` are between 0 - 255 and the minimum brightness difference will be 1, so I think this test can check if the choice by exact version is different from non-exact version. In fact, when I was mistakenly using `round` instead of `floor` in commit fff6afb, this test said errors.
But I found I should use EXPECT_LT in Line 176 and 180.

Reply:
When the matrix is [
[ 0, 1, 2, 3, ..., n ],
[ 0, 1, 2, 3, ..., n ],
[ 0, 1, 2, 3, ..., n ]
]
, we can compare the x-coordinate of scaled pixels.